documentation/swagger-good-patterns.md

```diff
@@ -13,10 +13,18 @@
  - Checks on the Resource Model
     - Every "Resource" Model should have "location" as a "required" property as per the mandate from ARM team; no exceptions are allowed over here.
     - Every "Resource" Model should be tagged with "x-ms-azure-resource". This will cause the Resource to inherit from the Resource definition in the client runtime.
+    - GET/PUT/PATCH for a resource should return the same "Resource" Model, also PUT should accept the same body (there can be readonly, optional, required properties).
  - Consistency in the naming of list methods
```

Comment:
It is worthwhile mentioning over here that the body parameter for a PATCH operation should have all optional properties and they should not have default values.